nexus/src/app/volume.rs

```diff
@@ -110,10 +112,27 @@ impl super::Nexus {
                 )
                 .await?;
 
+            // If the live repair or reconciliation was successfully completed,
+            // check if the repaired downstairs is part of a a region
```

Comment:
Nit "part of a a region" -> "part of a region"

Reply:
fixed in 269e8e667